docs/source/install.rst

```diff
@@ -1,15 +1,42 @@
 Installing
 ==========
 
-Here are two ways to hack on python.org:
+Here are three ways to hack on python.org:
 
-1. :ref:`vagrant-setup`
-2. :ref:`manual-setup`
+1. :ref:`docker-setup`
+2. :ref:`vagrant-setup`
+3. :ref:`manual-setup`
+
+.. _docker-setup:
+
+Docker Setup
+------------
+
+First, install Docker_ on your machine. The Docker installation will
+include docker-compose which can be used to bring up the application:
+
+::
+
+    $ docker-compose up
+
+docker-compose will bring up the application container and a PostgreSQL 
+container. The website will be available at http://localhost:8000.
+
+Each time changes are made to local files the application container
```

Comment:
I don't think this is true. We'd need to rebuild it if we'd updated dependencies, but we shouldn't have to rebuild it each time we changed Python files.

Reply:
My understanding is that the Dockerfile will copy the directory over on build (with any changes or updates) but if that's wrong I will remove this line.

Reply:
I'll check this out later. We are using Docker at $WORK and I don't remember if I had to rebuild it each time I made changes to the application code.